src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -186,7 +183,7 @@ export class Message extends React.Component<Props, any> {
 
             {this.renderBody()}
 
-            {!message.is_from_user && <FromSignature>{fromSignature}</FromSignature>}
+            {!message.is_from_user && <FromSignature>{fromName}</FromSignature>}
```

Comment:
Nice! I guess this can be null, in which case I this will render fine (div with height of 0), but maybe worth a check?

Overall 👍

Reply:
good point! ended up adding a new story for message and testing it there, noticed that not having from name was still adding space between messages, so ended up adding extra check for `fromName &&`. Now it looks like this:

![screen shot 2018-03-14 at 12 39 27 pm](https://user-images.githubusercontent.com/1230819/37400632-d03e0802-2785-11e8-8349-974cf1a4bd86.png)

and for a message with fromName:
![screen shot 2018-03-14 at 12 39 34 pm](https://user-images.githubusercontent.com/1230819/37400652-dac48c60-2785-11e8-8929-17c9a4d280d7.png)